mappings/net/minecraft/util/DismountVehicleUtils.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5275 net/minecraft/util/DismountVehicleUtils
```

Comment:
`entity/Dismounting` imo

Reply:
`entity/vehicle/Dismounting`?

Reply:
Does this apply to dismounting from any entity or just vehicles

Reply:
Either works for me